lib/main.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_projects/routes/movie_list.dart';
+import 'routes/movie_main_widget.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: MaterialApp(
+        theme: ThemeData.dark(),
+        initialRoute: '/listPreview',
+        routes: {
+          '/listPreview': (context) => const MovieList(),
+          '/MovieView': (context) => const MovieMainWidget(),
```

Comment:
By definition the initial path is always '/' and also move all the named routes to consts

Reply:
Done